profiles/models.py

```diff
@@ -99,6 +100,25 @@ class Profile(models.Model):
     mailing_address = models.TextField(blank=True, null=True)
     date_joined_micromasters = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     linkedin = JSONField(blank=True, null=True)
+    student_id = models.IntegerField(blank=True, null=True, unique=True)
+
+    @transaction.atomic
+    def save(self, *args, **kwargs):
+        """need to handle setting the student_id number"""
+        if self.id is None or self.student_id is None:
+            max_id = Profile.objects.aggregate(Max('student_id'))
+            if max_id['student_id__max'] is None:
+                max_id = 1
+            else:
+                max_id = max_id['student_id__max']
```

Comment:
you can write
```
if max_id['student_id__max'] is None:
    max_id = 1
else:
    max_id = max_id['student_id__max']
```
in a more pythonic way:
`max_id = max_id['student_id__max'] or 1`